docs/architecture.md

```diff
@@ -158,6 +160,7 @@ The design of the landing zone follows GCP best practices and architecture princ
 ### DI 0: GCP Secure Landing Zones - Best Practices
 
 ### DI 1: Decide on Shared VPC or Hub-and-spoke Network Topologies
+- see VPC network design in the KCC Landing Zone Architecture discussion - https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/dev/solutions/landing-zone/architecture.md#di-05-complete-network-design
```

Comment:
Something odd with this PR - patch - you seem to basing the diff on an older (pre Oct 2022) main branch
for example the change above for DI 1 - is already in under main
https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/blob/main/docs/architecture.md#di-1-decide-on-shared-vpc-or-hub-and-spoke-network-topologies

I think you need to merge main into your PR branch and regenerate your PR patch - to pickup changes in the last year

Reply:
Rebased.